### ISSUE TITLE
Mosaic attenuation pattern in lung

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7702]
**Preferred term label:**
Mosaic attenuation pattern in lung

**Synonyms**
Mosaic perfusion in lung

**Definition (free text, please give PubMed ID)**
Mosaic attenuation pattern in the lung is a term used to describe regions of differing pulmonary attenuation on CT imaging. It is a non-specific finding, although is associated with several conditions such as obstructive small airway disease, occlusive vascular disease (in which case is called mosaic perfusion pattern), pulmonary oedema, and parenchymal diseases.

**Parent term (use hpo.jax.org/app)**
HP:0031983

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Chronic thromboembolic pulmonary disease

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
There are
[Mosaic attenuation pattern on pulmonary HRCT HP:0025397](https://hpo.jax.org/app/browse/term/HP:0025397)
and
[Mosaic pulmonary attenuation pattern HP:0033537](https://hpo.jax.org/app/browse/term/HP:0033537)

I am not sure if these two terms are already synonyms. @eswietlik please advise

**Comment by eswietlik** [comment]
They are synonyms, sorry I must have missed them. No need to add this one.
All the best,
Emilia

**Comment by pnrobinson** [comment]
@eswietlik sorry for the long delay -- 2023 was the year of our relocation (back) to Germany.

Adding new term: 
- ID: HP:6000317
- label: Mosaic attenuation pattern in lung
- parents: [Abnormal pulmonary thoracic imaging finding (HP:0031983)](https://hpo.jax.org/app/browse/term/HP:0031983)
- pmid: 
- def: Mosaic attenuation pattern in the lung is a term used to describe regions of differing pulmonary attenuation on CT imaging. It is a non-specific finding, although is associated with several conditions such as obstructive small airway disease, occlusive vascular disease (in which case is called mosaic perfusion pattern), pulmonary oedema, and parenchymal diseases.
- comment: 
- orcid: orcid:0000-0002-4095-8489
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7702
- synonyms:
